src/transformers/models/albert/modeling_tf_albert.py

```diff
@@ -138,7 +138,7 @@ class TFAlbertEmbeddings(tf.keras.layers.Layer):
     def __init__(self, config: AlbertConfig, **kwargs):
         super().__init__(**kwargs)
 
-        self.vocab_size = config.vocab_size
+        self.config = config
         self.type_vocab_size = config.type_vocab_size
```

Comment:
if you have the patience to do it, removing the other attributes that are set from config (like `self.type_vocab_size`) would also be great :) That way there will be a greater similarity between PT and TF 🧡

Reply:
@gante 
Ok, I will remove `self.type_vocab_size`, btw there seem to be 3 more lines 
```
self.embedding_size = config.embedding_size
self.max_position_embeddings = config.max_position_embeddings
self.initializer_range = config.initializer_range
```
I don't think we need to change them for this purpose, right?

Reply:
It's not needed, but it would be a nice refactor :) Don't worry about it if you are not interested in making the change, we can pick it up afterward -- the `vocab_size` is the most important one for now, as it is causing bugs.